<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>startExam</class>
 <widget class="QMainWindow" name="startExam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>480</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>WatchU</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(252, 252, 244)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>360</width>
     <height>480</height>
    </size>
   </property>
   <widget class="QLabel" name="label_sn_title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>23</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquareRoundB</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>학       번 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_en_title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>시험번호 :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="finishButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>420</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="accessibleName">
     <string>startButton</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 230, 153)</string>
    </property>
    <property name="text">
     <string>시 험 종 료</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_sn_set">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>23</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_en_set">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="exam_ongoing_alert">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어_ac Bold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255, 0, 0, 40)</string>
    </property>
    <property name="text">
     <string>감시기능 작동 중</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="logList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>321</width>
      <height>231</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>321</width>
      <height>0</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
